hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/RawLocalFileSystem.java

```diff
@@ -137,8 +139,13 @@ public void initialize(URI uri, Configuration conf) throws IOException {
             STREAM_READ_SKIP_BYTES)
         .build();
 
+    /** Reference to the bytes read counter for slightly faster counting. */
+    private final AtomicLong bytesRead;
+
     public LocalFSFileInputStream(Path f) throws IOException {
       fis = new FileInputStream(pathToFile(f));
+      bytesRead = ioStatistics.getCounterReference(
```

Comment:
No issues here, just confirming if these changes aren't part of a different PR by mistake(IOStats PR).

Reply:
this is a followon to IOStats...it was actually blocked on the buffer between row local fs output and FSDataOutput:
`org.apache.hadoop.fs.statistics.BufferedIOStatisticsOutputStream` - needed for hsync passthrough to the File output stream. We're now generating IOStats for the local FS too.